src/components/InfoGerais.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+
+
+const styles= `
+    .BlocoInfoGerais{
+        border-radius: 50px;
+        width: 400px;
+        height: 150px;
+        border: 1px solid white;
+    }
+
+    .bg-orange {
+        background-color: #FF9551;
```

Comment:
```suggestion
        background-color: #F59489;
```
A cor da borda é diferente no Figma.

Reply:
Aaaaaah, eu tava olhando pra tela errada do Figma